README.rst

```diff
@@ -38,166 +70,83 @@ necessary to provide the most reliable results, but historically such PDFs
-
-See the `SrMise license <LICENSE.txt>`__ for terms and conditions of use.
-Detailed installation instructions for the `Windows`_, `Mac OS X`_, and
-`Linux`_ platforms follow.
-
-Windows
--------
+For more information about the diffpy.srmise library, please consult our `online documentation <https://diffpy.github.io/diffpy.srmise>`_.
 
-Several prebuilt Python distributions for Windows include all the
-prerequisite software required to run SrMise, and installing one of these is the
-simplest way to get started.  These distributions are usually free for
-individual and/or academic use, but some also have commercial version.  Links to
-executables, installation instructions, and licensing information
-for some popular options are listed below.
-
-* `Anaconda <https://www.anaconda.com/download>`_
-* `Enthought Canopy <https://www.enthought.com/products/canopy/>`_
-* `Python(x,y) <https://code.google.com/p/pythonxy/>`_
-* `WinPython <http://winpython.github.io>`_
-
-Alternately, individual Windows executables for Python and the required
-components can be downloaded and installed.  The official Windows releases of
-Numpy and SciPy do not currently support 64-bit Python installations, so be
-sure to download the 32-bit versions of these packages.
-
-* `Python 2.6/2.7 <https://www.python.org/downloads/windows/>`_
-* `NumPy <http://sourceforge.net/projects/numpy/files/NumPy/>`_
-* `SciPy <http://sourceforge.net/projects/scipy/files/scipy/>`_
-* `matplotlib <http://matplotlib.org/downloads.html>`_
-
-After installing Python and the required packages, the simplest way to obtain
-SrMise is using ``pip`` to download and install the latest release from the
-`Python Package Index <https://pypi.python.org>`_ (PyPI).  Open a command window
-by running ``cmd`` from the Start Menu's application search box (Windows 7/8/10)
-or Run command (Windows Vista and earlier).  Verify that the
-``pip`` program is installed by running ::
-
-    pip --version
-
-If this command is not found, download and run
-`get-pip.py <https://bootstrap.pypa.io/get-pip.py>`_, which will install both it
-and setuptools.  For example, if the file were downloaded to the desktop, a
-Windows user named ``MyName`` should run the following from the command
-line: ::
-
-    cd C:\Users\MyName\Desktop
-    python get-pip.py
-
-Finally, install the latest version of SrMise by running ::
-
-    pip install diffpy.srmise
-
-
-Mac OS X
+Citation
 --------
 
-For Mac OS X systems with the MacPorts package manager, the required
-software can be installed with ::
-
-   sudo port install \
-      python27 py27-setuptools py27-numpy py27-scipy py27-matplotlib
+If you use diffpy.srmise in a scientific publication, we would like you to cite this package as
 
-When installing for MacPorts, make sure the MacPorts bin directory is the first
-in the system PATH and that python27 is selected as the default Python version
-in MacPorts::
+        diffpy.srmise Package, https://github.com/diffpy/diffpy.srmise
```

Comment:
This one has a published paper to cite, so please replace (here and in the docs) this line with:
L. Granlund, Billinge, S. J. L., and P. M. Duxbury. “Algorithm for systematic peak extraction from atomic
pair distribution functions”. In: Acta Crystallogr. A 71.4 (2015), pp. 392–409. DOI:
10.1107/S2053273315005276.